cli/util/convert.go

```diff
@@ -74,7 +75,7 @@ func NewCommands() []*cli.Command {
 		&cli.UintFlag{
 			Name:  "index-file-size",
 			Usage: "Size of index file",
-			Value: 128000,
+			Value: neofs.DefaultIndexFileSize,
```

Comment:
Rename `neofs` to `blockstorage`. We already have `neofs` in oracle-related module.

Reply:
OK, I see that `neofs` is also moved from `services/oracle` to `services/neofs`. I'm not sure it's appropriate to keep `pkg/services/neofs` as a separate package, because it's not a fully-qualified service. This package is just a set of functions that interact with NeoFS and it also contains fetcher/uploader related constants. So it's more like a utility package. We need @roman-khimov to take this decision: where to place NeoFS-related helpers and common fetcher/uploader constants and helpers.

Reply:
@roman-khimov where do you see a proper place for neofs-related code and constants?

Reply:
Not `pkg/services/neofs`. Can be `pkg/internal/fsutil`. Can be `pkg/services/helpers/fsutil`. Something like that, depends on the contents as well.

Reply:
The problem is that for constants it's not quite obvious:
```
 fsutil.InitialBackoff
```
vs
```
 neofs.InitialBackoff
```
But probably even for that `neofs` looks like a better solution. OK, let's have `neofs`.